pkg/pillar/types/contenttreetypes.go

```diff
@@ -31,6 +31,8 @@ type ContentTreeConfig struct {
 	CustomMeta        string
 	// XXX DesignatedNodeID
 	DesignatedNodeID uuid.UUID
```

Comment:
A comment about its usage only in kubevirt could be helpful to have for others.

Reply:
ok.